### ISSUE TITLE
Error message when installing via `pip`

### ISSUE THREAD

**Issue by lucianoqueiroz** [localstack/awscli-local#31]
Hey, something is going on when installing `awscli-local` via `pip`:

```
docker run -ti --entrypoint bash python

root@dbb55d128863:/# pip install awscli-local
Collecting awscli-local
  Downloading awscli-local-0.12.tar.gz (5.0 kB)
Collecting localstack-client
  Downloading localstack-client-1.10.tar.gz (4.3 kB)
Collecting boto3
  Downloading boto3-1.16.40-py2.py3-none-any.whl (130 kB)
     |████████████████████████████████| 130 kB 1.8 MB/s
Collecting botocore<1.20.0,>=1.19.40
  Downloading botocore-1.19.40-py2.py3-none-any.whl (7.1 MB)
     |████████████████████████████████| 7.1 MB 15.5 MB/s
Collecting jmespath<1.0.0,>=0.7.1
  Downloading jmespath-0.10.0-py2.py3-none-any.whl (24 kB)
Collecting python-dateutil<3.0.0,>=2.1
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 11.9 MB/s
Collecting s3transfer<0.4.0,>=0.3.0
  Downloading s3transfer-0.3.3-py2.py3-none-any.whl (69 kB)
     |████████████████████████████████| 69 kB 8.5 MB/s
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting urllib3<1.27,>=1.25.4
  Downloading urllib3-1.26.2-py2.py3-none-any.whl (136 kB)
     |████████████████████████████████| 136 kB 27.7 MB/s
Building wheels for collected packages: awscli-local, localstack-client
  Building wheel for awscli-local (setup.py) ... done
  Created wheel for awscli-local: filename=awscli_local-0.12-py3-none-any.whl size=4529 sha256=5f0fb63be73b03ae5b4ca544438f7bc294c1d1ccc958eaefe6839a395728a665
  Stored in directory: /root/.cache/pip/wheels/ce/35/2f/26fb372ab125cb51548b95876ac844fda1c311455770e9b73c
  Building wheel for localstack-client (setup.py) ... done
  Created wheel for localstack-client: filename=localstack_client-1.10-py3-none-any.whl size=3809 sha256=8f76b99d892cc75c5cf02a97b61e3000011a4ded27392b71a88bc2b457c4998e
  Stored in directory: /root/.cache/pip/wheels/5e/ed/92/4f85adb3f58631a730c35d08bdad0fe7a6d0c48b8456e522f8
Successfully built awscli-local localstack-client
ERROR: Error while checking for conflicts. Please file an issue on pip's issue tracker: https://github.com/pypa/pip/issues/new
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3021, in _dep_map
    return self.__dep_map
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2815, in __getattr__
    raise AttributeError(attr)
AttributeError: _DistInfoDistribution__dep_map

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pip/_internal/commands/install.py", line 517, in _determine_conflicts
    return check_install_conflicts(to_install)
  File "/usr/local/lib/python3.9/site-packages/pip/_internal/operations/check.py", line 107, in check_install_conflicts
    would_be_installed = _simulate_installation_of(to_install, package_set)
  File "/usr/local/lib/python3.9/site-packages/pip/_internal/operations/check.py", line 135, in _simulate_installation_of
    package_set[name] = PackageDetails(dist.version, dist.requires())
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2736, in requires
    dm = self._dep_map
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3023, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3040, in _compute_dependencies
    common = frozenset(reqs_for_extra(None))
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3037, in reqs_for_extra
    if not req.marker or req.marker.evaluate({'extra': extra}):
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/markers.py", line 328, in evaluate
    return _evaluate_markers(self._markers, current_environment)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/markers.py", line 250, in _evaluate_markers
    groups[-1].append(_eval_op(lhs_value, op, rhs_value))
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/markers.py", line 201, in _eval_op
    return spec.contains(lhs)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/specifiers.py", line 206, in contains
    normalized_item = self._coerce_version(item)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/specifiers.py", line 170, in _coerce_version
    version = parse(version)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/version.py", line 57, in parse
    return Version(version)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/version.py", line 296, in __init__
    match = self._regex.search(version)
TypeError: expected string or bytes-like object
Installing collected packages: six, urllib3, python-dateutil, jmespath, botocore, s3transfer, boto3, localstack-client, awscli-local
Successfully installed awscli-local-0.12 boto3-1.16.40 botocore-1.19.40 jmespath-0.10.0 localstack-client-1.10 python-dateutil-2.8.1 s3transfer-0.3.3 six-1.15.0 urllib3-1.26.2

```

**Comment by microngust** [comment]
I'm seeing the exact same error. I'm running Python 3.9.1 on Ubuntu 20.04.1.

**Comment by lucianoqueiroz** [comment]
If you force the previous version the installation works fine

```
root@2f9ebb264380:/# pip install 'awscli-local==0.9'

Collecting awscli-local==0.9
  Downloading awscli-local-0.9.tar.gz (4.7 kB)
Collecting awscli
  Downloading awscli-1.18.200-py2.py3-none-any.whl (3.5 MB)
     |████████████████████████████████| 3.5 MB 1.5 MB/s
Collecting botocore==1.19.40
  Downloading botocore-1.19.40-py2.py3-none-any.whl (7.1 MB)
     |████████████████████████████████| 7.1 MB 1.3 MB/s
Collecting colorama<0.4.4,>=0.2.5
  Downloading colorama-0.4.3-py2.py3-none-any.whl (15 kB)
Collecting docutils<0.16,>=0.10
  Downloading docutils-0.15.2-py3-none-any.whl (547 kB)
     |████████████████████████████████| 547 kB 2.4 MB/s
Collecting jmespath<1.0.0,>=0.7.1
  Downloading jmespath-0.10.0-py2.py3-none-any.whl (24 kB)
Collecting python-dateutil<3.0.0,>=2.1
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 2.2 MB/s
Collecting PyYAML<5.4,>=3.10
  Downloading PyYAML-5.3.1.tar.gz (269 kB)
     |████████████████████████████████| 269 kB 1.8 MB/s
Collecting rsa<=4.5.0,>=3.1.2
  Downloading rsa-4.5-py2.py3-none-any.whl (36 kB)
Collecting pyasn1>=0.1.3
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
     |████████████████████████████████| 77 kB 1.1 MB/s
Collecting s3transfer<0.4.0,>=0.3.0
  Downloading s3transfer-0.3.3-py2.py3-none-any.whl (69 kB)
     |████████████████████████████████| 69 kB 1.5 MB/s
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting urllib3<1.27,>=1.25.4
  Downloading urllib3-1.26.2-py2.py3-none-any.whl (136 kB)
     |████████████████████████████████| 136 kB 1.8 MB/s
Collecting localstack-client
  Downloading localstack-client-1.10.tar.gz (4.3 kB)
Collecting boto3
  Downloading boto3-1.16.40-py2.py3-none-any.whl (130 kB)
     |████████████████████████████████| 130 kB 2.1 MB/s
Building wheels for collected packages: awscli-local, PyYAML, localstack-client
  Building wheel for awscli-local (setup.py) ... done
  Created wheel for awscli-local: filename=awscli_local-0.9-py3-none-any.whl size=4285 sha256=6c44b91dac5031a03329b6e4baa9401bdb4c1fd2c83ff75428f4fcdfed8baf8b
  Stored in directory: /root/.cache/pip/wheels/2e/a2/18/ce42fd22a4aaf47732f00a85f22b292d59f782fe7c3aa33da6
  Building wheel for PyYAML (setup.py) ... done
  Created wheel for PyYAML: filename=PyYAML-5.3.1-cp39-cp39-linux_x86_64.whl size=542382 sha256=ccd6e73a38ee652136c34b404157de2422cd3732291a9c92311ed77ba0da5758
  Stored in directory: /root/.cache/pip/wheels/69/60/81/5cd74b8ee068fbe9e04ca0d53148f28f5c6e2c5b177d5dd622
  Building wheel for localstack-client (setup.py) ... done
  Created wheel for localstack-client: filename=localstack_client-1.10-py3-none-any.whl size=3809 sha256=686a2a1367d59626c0ed2c36191be1266e7df51e0d8a45800868372bc3733146
  Stored in directory: /root/.cache/pip/wheels/5e/ed/92/4f85adb3f58631a730c35d08bdad0fe7a6d0c48b8456e522f8
Successfully built awscli-local PyYAML localstack-client
Installing collected packages: six, urllib3, python-dateutil, jmespath, botocore, s3transfer, pyasn1, rsa, PyYAML, docutils, colorama, boto3, localstack-client, awscli, awscli-local
Successfully installed PyYAML-5.3.1 awscli-1.18.200 awscli-local-0.9 boto3-1.16.40 botocore-1.19.40 colorama-0.4.3 docutils-0.15.2 jmespath-0.10.0 localstack-client-1.10 pyasn1-0.4.8 python-dateutil-2.8.1 rsa-4.5 s3transfer-0.3.3 six-1.15.0 urllib3-1.26.2

```

**Comment by lucianoqueiroz** [comment]
looks like `pip install awscli-local[v1]` also works:

```
root@15cd78cb0c30:/# pip install awscli-local[v1]
Collecting awscli-local[v1]
  Downloading awscli-local-0.12.tar.gz (5.0 kB)
Collecting awscli
  Downloading awscli-1.18.200-py2.py3-none-any.whl (3.5 MB)
     |████████████████████████████████| 3.5 MB 123 kB/s
Collecting botocore==1.19.40
  Downloading botocore-1.19.40-py2.py3-none-any.whl (7.1 MB)
     |████████████████████████████████| 7.1 MB 131 kB/s
Collecting colorama<0.4.4,>=0.2.5
  Downloading colorama-0.4.3-py2.py3-none-any.whl (15 kB)
Collecting docutils<0.16,>=0.10
  Downloading docutils-0.15.2-py3-none-any.whl (547 kB)
     |████████████████████████████████| 547 kB 209 kB/s
Collecting jmespath<1.0.0,>=0.7.1
  Downloading jmespath-0.10.0-py2.py3-none-any.whl (24 kB)
Collecting python-dateutil<3.0.0,>=2.1
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 202 kB/s
Collecting PyYAML<5.4,>=3.10
  Downloading PyYAML-5.3.1.tar.gz (269 kB)
     |████████████████████████████████| 269 kB 122 kB/s
Collecting rsa<=4.5.0,>=3.1.2
  Downloading rsa-4.5-py2.py3-none-any.whl (36 kB)
Collecting pyasn1>=0.1.3
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
     |████████████████████████████████| 77 kB 258 kB/s
Collecting s3transfer<0.4.0,>=0.3.0
  Downloading s3transfer-0.3.3-py2.py3-none-any.whl (69 kB)
     |████████████████████████████████| 69 kB 368 kB/s
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting urllib3<1.27,>=1.25.4
  Downloading urllib3-1.26.2-py2.py3-none-any.whl (136 kB)
     |████████████████████████████████| 136 kB 191 kB/s
Collecting localstack-client
  Downloading localstack-client-1.10.tar.gz (4.3 kB)
Collecting boto3
  Downloading boto3-1.16.40-py2.py3-none-any.whl (130 kB)
     |████████████████████████████████| 130 kB 264 kB/s
Building wheels for collected packages: awscli-local, PyYAML, localstack-client
  Building wheel for awscli-local (setup.py) ... done
  Created wheel for awscli-local: filename=awscli_local-0.12-py3-none-any.whl size=4529 sha256=a614a578d2434c87b1c85c1b55a802c46331fab193d2c5375931d708eeafd062
  Stored in directory: /root/.cache/pip/wheels/ce/35/2f/26fb372ab125cb51548b95876ac844fda1c311455770e9b73c
  Building wheel for PyYAML (setup.py) ... done
  Created wheel for PyYAML: filename=PyYAML-5.3.1-cp39-cp39-linux_x86_64.whl size=542360 sha256=2c6f65c4e79b2c5bf4a9192b09d6fd78bdb877b052dfe50cf600b6496e7c5a5d
  Stored in directory: /root/.cache/pip/wheels/69/60/81/5cd74b8ee068fbe9e04ca0d53148f28f5c6e2c5b177d5dd622
  Building wheel for localstack-client (setup.py) ... done
  Created wheel for localstack-client: filename=localstack_client-1.10-py3-none-any.whl size=3809 sha256=24e7f86d6c28fb95ee582e98d4a50e8c4697a45cdfeadd02bf24bd9441e0ccf4
  Stored in directory: /root/.cache/pip/wheels/5e/ed/92/4f85adb3f58631a730c35d08bdad0fe7a6d0c48b8456e522f8
Successfully built awscli-local PyYAML localstack-client
ERROR: Error while checking for conflicts. Please file an issue on pip's issue tracker: https://github.com/pypa/pip/issues/new
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3021, in _dep_map
    return self.__dep_map
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2815, in __getattr__
    raise AttributeError(attr)
AttributeError: _DistInfoDistribution__dep_map

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pip/_internal/commands/install.py", line 517, in _determine_conflicts
    return check_install_conflicts(to_install)
  File "/usr/local/lib/python3.9/site-packages/pip/_internal/operations/check.py", line 107, in check_install_conflicts
    would_be_installed = _simulate_installation_of(to_install, package_set)
  File "/usr/local/lib/python3.9/site-packages/pip/_internal/operations/check.py", line 135, in _simulate_installation_of
    package_set[name] = PackageDetails(dist.version, dist.requires())
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 2736, in requires
    dm = self._dep_map
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3023, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3040, in _compute_dependencies
    common = frozenset(reqs_for_extra(None))
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/pkg_resources/__init__.py", line 3037, in reqs_for_extra
    if not req.marker or req.marker.evaluate({'extra': extra}):
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/markers.py", line 328, in evaluate
    return _evaluate_markers(self._markers, current_environment)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/markers.py", line 250, in _evaluate_markers
    groups[-1].append(_eval_op(lhs_value, op, rhs_value))
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/markers.py", line 201, in _eval_op
    return spec.contains(lhs)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/specifiers.py", line 206, in contains
    normalized_item = self._coerce_version(item)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/specifiers.py", line 170, in _coerce_version
    version = parse(version)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/version.py", line 57, in parse
    return Version(version)
  File "/usr/local/lib/python3.9/site-packages/pip/_vendor/packaging/version.py", line 296, in __init__
    match = self._regex.search(version)
TypeError: expected string or bytes-like object
Installing collected packages: six, urllib3, python-dateutil, jmespath, botocore, s3transfer, pyasn1, boto3, rsa, PyYAML, localstack-client, docutils, colorama, awscli-local, awscli
Successfully installed PyYAML-5.3.1 awscli-1.18.200 awscli-local-0.12 boto3-1.16.40 botocore-1.19.40 colorama-0.4.3 docutils-0.15.2 jmespath-0.10.0 localstack-client-1.10 pyasn1-0.4.8 python-dateutil-2.8.1 rsa-4.5 s3transfer-0.3.3 six-1.15.0 urllib3-1.26.2
```

It prints the same error message though.

**Comment by rdmrocha** [comment]
I can confirm that this issue was introduced with 0.12

Facing the exact same thing and it's not present if rolling back to 0.11

**Comment by whummer** [comment]
Thanks for reporting @lucianoqueiroz @microngust . Looks like this was related to an incompatibility with some newer `pip` versions - the issue should be fixed in the meantime, new version `0.13` has been released. Please report here if the problem persists. Thanks
